server/src/test/java/io/crate/role/metadata/UsersPrivilegesMetadataTest.java

```diff
@@ -68,8 +69,13 @@ static UsersPrivilegesMetadata createMetadata() {
         }
         usersPrivileges.put(USER_WITHOUT_PRIVILEGES, new HashSet<>());
         usersPrivileges.put(USER_WITH_DENIED_DQL, new HashSet<>(Collections.singletonList(DENY_DQL)));
-        usersPrivileges.put(USER_WITH_SCHEMA_AND_TABLE_PRIVS, new HashSet<>(
-            Arrays.asList(GRANT_SCHEMA_DML, GRANT_TABLE_DQL, GRANT_TABLE_DDL, GRANT_VIEW_DQL, GRANT_VIEW_DML, GRANT_VIEW_DDL)));
+        usersPrivileges.put(USER_WITH_TABLE_VIEW_SCHEMA_AL_PRIVS, new HashSet<>(
```

Comment:
How is this change related?

Reply:
This test is about the old User Metadata, and it imports the same users from the other test. Since it's something to be removed in the future, I didn't spend time to separate them.